wikimedia/wikipedia-ios
# improve TestFlight messages

**Phabricator:** N/A

### Notes
* Tightens up the TestFlight message, and adds a detailed message to the git tag.

### Test Steps
1. Using Jenkins, deploy app (https://wmf2249:8043/job/Deploy/) and choose this branch. 
2. Ensure the TestFlight text looks nice.

(Alt: See output from https://wmf2249:8043/job/Deploy/401/ below.)

### Photos
![Screenshot](https://user-images.githubusercontent.com/9295855/121605576-7fa17d80-ca01-11eb-8230-779008f186b5.png)
![IMG_0DAC95BB883F-1](https://user-images.githubusercontent.com/9295855/121605571-7ca68d00-ca01-11eb-98c0-ee7de1049438.jpeg)
